packages/airnode-protocol/credentials.example.json

```diff
@@ -9,6 +9,7 @@
       "sepolia": "",
       "arbitrumOne": "<arbitrum>",
       "arbitrumTestnet": "<arbitrum-testnet>",
+      "aurora": "<aurora>",
```

Comment:
This is added but I'm seeing that the contracts are not verified on https://aurorascan.dev/

Reply:
hardhat does support aurora but when try to run the verify command I get the following error:
```
airnode-protocol/deploy/2_verify.js:
: ENOENT: no such file or directory, open '/home/ashar/API3/airnode/packages/airnode-protocol/artifacts/build-info/37ae4407f650f87f7cba868b566e365f.json'
    at DeploymentsManager.executeDeployScripts (/home/ashar/API3/airnode/node_modules/hardhat-deploy/src/DeploymentsManager.ts:1222:19)
    at DeploymentsManager.runDeploy (/home/ashar/API3/airnode/node_modules/hardhat-deploy/src/DeploymentsManager.ts:1052:5)
    at SimpleTaskDefinition.action (/home/ashar/API3/airnode/node_modules/hardhat-deploy/src/index.ts:438:5)
    at Environment._runTaskDefinition (/home/ashar/API3/airnode/node_modules/hardhat/src/internal/core/runtime-environment.ts:219:14)
    at Environment.run (/home/ashar/API3/airnode/node_modules/hardhat/src/internal/core/runtime-environment.ts:131:14)
    at SimpleTaskDefinition.action (/home/ashar/API3/airnode/node_modules/hardhat-deploy/src/index.ts:584:32)
    at Environment._runTaskDefinition (/home/ashar/API3/airnode/node_modules/hardhat/src/internal/core/runtime-environment.ts:219:14)
    at Environment.run (/home/ashar/API3/airnode/node_modules/hardhat/src/internal/core/runtime-environment.ts:131:14)
    at SimpleTaskDefinition.action (/home/ashar/API3/airnode/node_modules/hardhat-deploy/src/index.ts:669:5)
    at Environment._runTaskDefinition (/home/ashar/API3/airnode/node_modules/hardhat/src/internal/core/runtime-environment.ts:219:14)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.Error:: command not found
```

Reply:
The contracts do get verified locally

Reply:
I had all previous deployments verified on the etherscan/blockscout block explorers manually. You can find the flattened versions at ftmscan.com or flatten them yourself using `hardhat flatten`.

Reply:
This is needed for decoding of transactions and in the etherscan case to allow the user interact with the contract over etherscan, it's not necessarily about verification that the contract is correct (that's what deterministic deployments and `verify-local` are for).